piet-common/src/cairo_back.rs

```diff
@@ -121,4 +126,19 @@ impl<'a> BitmapTarget<'a> {
         }
         Ok(raw_data)
     }
+
+    pub fn save_to_file(self, path: &str) -> Result<(), piet::Error> {
+        let height = self.surface.get_height();
+        let width = self.surface.get_width();
+        let image = self.into_raw_pixels(ImageFormat::RgbaPremul)?;
+        let file = BufWriter::new(File::create(path)
+            .map_err(|e| Into::<Box<dyn std::error::Error>>::into(e))?);
```

Comment:
I was playing around trying to figure out if there was a less verbose way to do this conversion, and the best I can come up with is,

```rust
.map_err(|e| Box::new(e) as Box<_>)?;
// also works:
.map_err(|e| Into::<Box<_>>::into(e))?;
```
In any case this is fine, I was just playing around.

Reply:
That truly looks better